### DISCUSSION TITLE
docs/api/translate/openai

### DISCUSSION THREAD

**Discussion by giscus** [STranslate/STranslate#144] [Announcements]
# docs/api/translate/openai

WPF 开发的一款即开即用、即用即走的翻译(OCR)工具

https://stranslate.zggsong.com/docs/api/translate/openai.html

**Comment by xinxuezi** [comment]
你好，请问一下，这个openai这个密钥具体应该这么做呀？

**Reply by ZGGSONG** [reply]
我自己当时开发去tb买了一个，目前也还在使用 [AiCore API(包含我个人的推广链接)](https://oneapi.xty.app/register?aff=V3qp) 申请到了key到设置-翻译服务-添加`OpenAI`服务-填入APIUrl、模型和API-Key即可

**Comment by yumkea** [comment]
伟大，无需多言!
